pkg/controller/v1beta1/inferenceservice/utils/utils.go

```diff
@@ -85,18 +84,35 @@ func IsMemoryResourceAvailable(isvc *v1beta1api.InferenceService, totalReqMemory
 	return predictorMemoryLimit.Cmp(totalReqMemory) >= 0
 }
 
+func getModelNameFromArgs(args []string) string {
+	for i, arg := range args {
+		if strings.HasPrefix(arg, constants.ArgumentModelName) {
+			// Case 1: ["--model-name=<model-name>"]
+			modelNameValueArr := strings.Split(arg, "=")
+			if len(modelNameValueArr) == 2 {
+				return modelNameValueArr[1]
+			}
+			// Case 2: ["--model-name <model-name>"]
```

Comment:
Do you want to tolerate more than one space in between? If so, you may need something more complex than a Split.

Reply:
Addressed.